docs/source/releases/latest/20-update-pyproject-depedencies.yaml

```diff
@@ -0,0 +1,35 @@
+bug fix:
+- description: |
+    Fixed a small bug in template_basic_plugin's pyproject.toml. The file was incorrectly set up after the switch to poetry and my vscode TOML highlighter caught the error with the dependency.
+  files:
+    added:
+    - ''
+    deleted:
+    - ''
+    modified:
+    - 'pyproject.toml'
+    moved:
+    - ''
+  related-issue:
+    number: 20
+    repo_url: ''
+  title: 'Fixed error in template_basic_plugin pyproject.toml'
+documentation:
+- description: 'Add a ``latest`` directory and add a yaml release note.'
+  files:
+    added:
+    - 'docs/source/release/latest'
+    - 'docs/source/release/latest/20-update-pyproject-depedencies.yaml'
+    deleted:
+    - ''
+    modified:
+    - tests/scripts/amsr2.global_clean.89-PCT-Using-Product-Defaults.sh
+    - tests/scripts/amsr2.tc_clean.89-PCT-Fully-Specified.sh
+    - tests/scripts/test_config.sh
+    moved:
+    - ''
+  related-issue:
+    number: 27
+    repo_url: ''
+  title: 'Set up repo for release notes built via brassy'
```

Comment:
I think this section can/should be removed.  I don't think release notes need to reference themselves.  No more release notes in release notes!

Reply:
Actually, it should be updated so it is ONLY the testing updates (ie, remove all the stuff about release notes, and make that section just that the test scripts were updated for "geoips run single_source")

Reply:
I will change this!

Reply:
Fixed!